src/containers/tutor-home-page/general-info-step/GeneralInfoStep.jsx

```
@@ -15,15 +15,21 @@
 
 import useForm from '~/hooks/use-form'
 import useDebounce from '~/hooks/use-debounce'
 import { nameField, textField } from '~/utils/validations/common'
 
 const GeneralInfoStep = ({ btnsBox }) => {
   const { stepData, handleStepData } = useStepContext()
+  const { setUnsavedChanges } = useModalContext()
+  const { setNeedConfirmation } = useConfirm()
   const contextData = stepData.generalInfo.data
   const contextErrors = stepData.generalInfo.errors
+  const { firstName, lastName, city, country, professionalSummary } =
```

ESLint

[error] 'firstName' is assigned a value but never used.

(no-unused-vars)